docs/source/projects.rst

```diff
@@ -446,7 +446,7 @@ You can run your MLflow Project on Kubernetes by following these steps:
    - ``kube-context``
      The `Kubernetes context
      <https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#context>`_
-     where MLflow will run the job.
+     where MLflow will run the job. If not provided, MLflow will assume it is running in a Kubernetes cluster.
```

Comment:
In the case where `kube-context` is unspecified, can you clarify that the in-cluster configuration will be selected and that this will use the Kubernetes service account configured in the current pod?